nautobot/core/templates/utilities/render_jinja2.html

```diff
@@ -0,0 +1,117 @@
+{% extends "base.html" %}
+{% load form_helpers %}
+{% load helpers %}
+{% load static %}
+
+{% block extra_styles %}
+<style type="text/css">
+    .button-container {
+        margin-bottom: 24px;
+    }
+    #render_jinja_template_form textarea {
+        resize: vertical;
+    }
+
+</style>
+{% endblock extra_styles %}
+
+{% block header %}
+<h1>{% block title %}Jinja Template Renderer{% endblock %}</h1>
+<hr>
+{% endblock %}
+
+
+
+{% block content %}
+<form class="form form-horizontal" onsubmit="handleFormSubmit(event)" id="render_jinja_template_form">
+    <div class="row">
+        <div class="col-lg-6 col-md-6">
+            <div class="panel panel-default">
+                <div class="panel-heading"><strong>Template</strong></div>
+                <div class="panel-body">
+                    <div class="form-group">
+                        <label class="col-md-3 control-label required" for="id_template_code">Jinja Template</label>
+                        <div class="col-md-9">
+                            <textarea cols="40" rows="10" class="form-control" id="id_template_code" required=""></textarea>
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label class="col-md-3 control-label" for="id_context">Data</label>
+                        <div class="col-md-9">
+                            <textarea cols="40" rows="10" class="form-control" placeholder="{}" id="id_context">{}</textarea>
+                            <span class="help-block">Enter data in <a href="https://json.org/">JSON</a> format.</span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="button-container text-right">
+                <button type="submit" class="btn btn-primary">Render</button>
+                <a href="{{ return_url }}" class="btn btn-default">Cancel</a>
```

Comment:
Where's the `return_url` coming from? Honestly it feels a bit weird to me to have a Cancel button on this page at all?

Reply:
I'm ok with removing it completely